packages/core/src/components/MonthPicker/MonthPicker.jsx

```diff
@@ -0,0 +1,220 @@
+    if (!this.isControlled) {
+      this.setState({ selectedMonths: this.monthsArray(false) });
+    }
+  }
+
+  renderMonths(selectedMonths) {
+    const disabledMonths = this.disabledMonths();
+    const { name, inversed } = this.props;
+    return (
+      <ul className="ds-c-list--bare ds-u-display--flex ds-u-justify-content--between ds-u-flex-wrap--wrap">
+        {this.months.map((month, i) => (
+          <Choice
+            name={name}
+            value={i}
+            checked={selectedMonths[i]}
+            key={month}
+            onChange={e => this.handleChange(e)}
+            className="ds-c-month-picker__month"
+            disabled={disabledMonths[i]}
+            inversed={inversed}
+            aria-label={this.monthsLong[i]}
+          >
+            {month}
+          </Choice>
+        ))}
+      </ul>
+    );
+  }
+
+  renderButton(text, onClick) {
+    return (
+      <Button
+        variation="primary"
+        size="small"
+        className="ds-u-margin-right--1"
+        onClick={onClick}
+        inversed={this.props.inversed}
+      >
+        {text}
+      </Button>
+    );
+  }
+
+  render() {
+    const { selectAllText, clearAllText } = this.props;
+    return (
+      <div className="ds-c-month-picker ds-u-margin-y--3">
+        <div className="ds-u-margin-bottom--3">
+          {this.renderButton(selectAllText, () => this.handleSelectAll())}
+          {this.renderButton(clearAllText, () => this.handleClearAll())}
+        </div>
+        {this.renderMonths(this.selectedMonths())}
+      </div>
+    );
+  }
+}
+
+MonthPicker.propTypes = {
+  /**
+   * The `input` field's `name` attribute
+   */
+  name: PropTypes.string.isRequired,
+  /**
+   * Locale string used to generate month names
+   */
+  locale: PropTypes.string,
+  /**
+   * Applies the "inverse" UI theme
+   */
+  inversed: PropTypes.bool,
+  /**
+   * Array of 12 booleans representing 12 months, where a value of `true`
+   * means the corresponding month is disabled.
+   */
+  disabledMonths: PropTypes.arrayOf(PropTypes.bool),
+  /**
+   * Array of 12 booleans representing 12 months, where a value of `true`
+   * means the corresponding month is selected.
+   * This will render a read-only field. If the field should be mutable,
+   * use `defaultSelectedMonths`.
+   */
+  selectedMonths: PropTypes.arrayOf(PropTypes.bool),
+  /**
+   * Array of 12 booleans representing 12 months, where a value of `true`
+   * means the corresponding month is selected by default.
+   * Sets the initial checked state for the 12 month checkboxes. Use this for
+   * an uncontrolled component; otherwise, use the `selectedMonths` property.
+   */
+  defaultSelectedMonths: PropTypes.arrayOf(PropTypes.bool),
+  onChange: PropTypes.func,
+  onSelectAll: PropTypes.func,
+  onClearAll: PropTypes.func,
+  /**
+   * For internationalization purposes, the text for the "Select all"
+   * button must be passed in as a prop.
+   */
+  selectAllText: PropTypes.string.isRequired,
```

Comment:
We could set an initial value for this in `defaultProps` to support prototyping without i18n.  (See [`DateField.defaultProps`](https://github.com/CMSgov/design-system/blob/master/packages/core/src/components/DateField/DateField.jsx#L104-L114) for an example of how we're doing this in other components).

Reply:
I didn't know we were doing this as it could cause users to forget that it has to be translated

Reply:
We should definitely encourage and support localization, but I don't think we should require it. I think it's okay to have a sane default when we're able to.